front/app/containers/CustomPageShow/CustomPageHeader/index.tsx

```diff
@@ -21,6 +22,9 @@ const CustomPageHeader = ({ pageData }: Props) => {
       {pageAttributes.banner_layout === 'two_row_layout' && (
         <TwoRowLayout pageData={pageData} />
       )}
+      {pageAttributes.banner_layout === 'fixed_ratio_layout' && (
```

Comment:
Unit test - does this component show when it should. Bonus, if we check the other layouts, as well.